src/Appwrite/Platform/Workers/Webhooks.php

```diff
@@ -47,8 +51,8 @@ public function action(Message $message): void
         $user = new Document($payload['user'] ?? []);
 
         foreach ($project->getAttribute('webhooks', []) as $webhook) {
-            if (array_intersect($webhook->getAttribute('events', []), $events)) {
-                $this->execute($events, $webhookPayload, $webhook, $user, $project);
+            if ($webhook->getAttribute('enabled') === true && array_intersect($webhook->getAttribute('events', []), $events)) {
```

Comment:
This check is here, and also on line 75. Lets keep it in execute() only. We can remove it from here